js/views/userPage/UserPageFollow.js

```diff
@@ -1,23 +1,56 @@
-import loadTemplate from '../../utils/loadTemplate';
-import { View } from 'backbone';
+import BaseVw from '../baseVw';
+import userShort from '../userShort';
+import app from '../../app';
+import Follows from '../../collections/UsersShort';
 
-export default class extends View {
+export default class extends BaseVw {
   constructor(options = {}) {
-    super({
-      className: 'userPageFollow',
-      events: {
-      },
-      ...options,
+    super(options);
+    this.options = options;
+    this.followType = options.followType;
+
+    if (!options.ownPage) {
+      this.followCol = new Follows(null, {
+        type: this.followType.toLowerCase(),
+        guid: this.model.id,
+      });
+      this.followCol.fetch().done(() => {
+        this.folRender();
+      });
+    } else {
+      this.followCol = app[`own${this.followType}`];
+      this.folRender();
+    }
+
+    this.listenTo(app[`own${this.followType}`], 'sync, update', () => {
+      this.folRender();
     });
   }
 
-  render() {
-    loadTemplate('userPage/userPageFollow.html', (t) => {
-      this.$el.html(t({
-        ...this.model.toJSON(),
-      }));
-    });
+  className() {
+    return 'userPageFollow flexRow';
+  }
 
+  folRender() {
```

Comment:
This is a leftover, originally I had a template that was rendered and the folRender added to it.